source/databases-collections.txt

```diff
@@ -0,0 +1,273 @@
+
+.. code-block:: python
+   :emphasize-lines: 2
+
+   database = client["test_database"]
+   database.create_collection("example_collection")
+
+You can specify collection options, such as maximum size and document
+validation rules, by passing them in as keyword arguments. For a full list of
+optional parameters, see the `create_collection() API documentation
+<{+api-root+}pymongo/database.html#pymongo.database.Database.create_collection>`__.
+
+Get a List of Collections
+-------------------------
+
+You can query for a list of collections in a database by calling the
+``list_collections()`` method. The method returns a cursor containing all
+collections in the database and their associated metadata.
+
+The following example calls the ``list_collections()`` method and iterates over
+the cursor to print the results:
+
+.. code-block:: python
+
+   collection_list = database.list_collections()
+
+   for c in collection_list:
+       print(c)
+
+To query for only the names of the collections in the database, call the
+``list_collection_name()`` method as follows:
+
+.. code-block:: python
+
+   collection_list = database.list_collection_names()
+
+   for c in collection_list:
+       print(c)
+
+For more information about iterating over a cursor, see :ref:`pymongo-cursors`. 
+
+Delete a Collection
+-------------------
+
+You can delete a collection from the database by using the ``drop_collection()``
+method.
+
+The following example deletes the ``test_collection`` collection:
+
+.. code-block:: python
+
+   collection = database["test_collection"];
+   collection.drop();
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database permanently deletes all
+   documents and all indexes within that collection.
+
+   Drop a collection only if the data in it is no longer needed.
+
+Configure Read and Write Operations
+-----------------------------------
+
+You can control how the driver routes read operations by setting a **read preference**.
+You can also control options for how the driver waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
+
+By default, databases inherit these settings from the ``MongoClient`` instance,
+and collections inherit them from the database. However, you can change these
+settings on your database or collection by using one of the following methods:
+
+- ``get_database()``: Gets the database and applies the client's read
+   preference, read concern, and write preference.
+- ``database.with_options()``: Gets the database and applies its current read
+   preference, read concern, and write preference.
+- ``get_collection()``: Gets the collection and applies its current read preference, read concern, and write preference.
+- ``collection.with_options()``: Gets the collection and applies the database's read
+   preference, read concern, and write preference.
```

Comment:
These are rendering with extra bold text. Is there an extra space on the second line of each item?